templates/settings.php

```diff
@@ -108,6 +111,56 @@
           >
         </td>
       </tr>
+      <?php if ($product_attributes): ?>
+        <tr class="form-field">
```

Comment:
What are those options for? Seems this options are responsible for the tracking only but do not affect the actual slugs?

Reply:
This options allows to select which attribute is the one used to assign brands/categories to a product. Then I can use that attribute slug to retrieve the product brand/category and add it to the tracking data set.

The slugs are not modified.